contributing/community/speaker-mentoring.rst

```diff
@@ -0,0 +1,43 @@
+Speaker mentoring
+=================
+
+The Symfony community benefits greatly from as many people as possible
+sharing their knowledge and experience with others. Every different
+point of view adds to our collective understanding of how to best use
+and evolve the code, design patterns and architecture provided within
+the Symfony community. As such we specifically welcome not only sharing
+from long time contributors, but also from new users as they will come
+across entirely different challenges with a totally fresh new look and
+perspective.
+
+How to get started
+------------------
+
+Giving a first talk at a conference can seem quite intimidating but we
+want to make sure that as many people as possible are empowered to take
+this path if they are motivated. We have collected a few resources with
+advice to get started. More importantly we connect experienced speakers
+with people just taking their first steps in this area:
+
+.. tip::
+
+    A good first step might be giving a talk at a local user group to a
+    smaller crowd that one knows more intimately. As a next step give
+    a talk at conference in your mother tongue.
```

Comment:
As a next step give a talk at **a** conference in your mother tongue.

Reply:
These two sentences aren't constructed in the same way. I would change the second one to "A next step could be giving a talk at a conference...".

Reply:
Taking into account the comment of @weaverryan on the previous sentence, this sentence would become "A next step could be to give a talk at a conference...".